<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Switches" name="SW1" native="true">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>60</y>
      <width>100</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>80</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>debug area</string>
    </property>
   </widget>
   <widget class="QLabel" name="debug_1">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>110</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="debug_2">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>130</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="debug_3">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>150</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="debug_4">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>170</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="debug_5">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>190</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="debug_7">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>230</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="debug_6">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>210</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="debug_8">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>250</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="debugJ_3">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>350</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="debugJ_4">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>370</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="debugJ_5">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>390</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="debugJ_6">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>410</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="debugJ_7">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>430</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="debugJ_2">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>330</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="debugJ_1">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>310</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="debugJ_8">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>450</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="J1_J9" name="j1_j9" native="true">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>20</y>
      <width>70</width>
      <height>800</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="debugJ_9">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>470</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="Buttons" name="buttons" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>410</y>
      <width>450</width>
      <height>320</height>
     </rect>
    </property>
   </widget>
   <widget class="Diodes" name="diodes" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>350</width>
      <height>320</height>
     </rect>
    </property>
   </widget>
   <widget class="MCU" name="mcu" native="true">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>0</y>
      <width>150</width>
      <height>800</height>
     </rect>
    </property>
   </widget>
   <widget class="Switches" name="SW2" native="true">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>60</y>
      <width>100</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="Switches" name="SW3" native="true">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>260</y>
      <width>100</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="Switches" name="SW4" native="true">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>260</y>
      <width>100</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="PORTS" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>20</y>
      <width>60</width>
      <height>800</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Switches</class>
   <extends>QWidget</extends>
   <header>switches.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>J1_J9</class>
   <extends>QWidget</extends>
   <header>j1_j9.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Buttons</class>
   <extends>QWidget</extends>
   <header>buttons.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Diodes</class>
   <extends>QWidget</extends>
   <header>diodes.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MCU</class>
   <extends>QWidget</extends>
   <header>mcu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PORTS</class>
   <extends>QWidget</extends>
   <header>ports.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
